src/core/generators/verbsOptions.ts

```diff
@@ -5,6 +5,7 @@ import {
   PathItemObject,
   ReferenceObject,
 } from 'openapi3-ts';
+import { keyword } from 'esutils';
```

Comment:
you also need to update the esutils.d.ts to add the type of the function

Reply:
Fixing.